### ISSUE TITLE
Formatting axis labels

### ISSUE THREAD

**Issue by sntgluca** [bloomberg/bqplot#957]
I wish to be able to control the axis label style and format, such as font properties, in a similar way we do with ticks.

I don't mind too much about the API, but something like the following seems reasonable to me:
```
bq.Axis(label='my label', label_format={'font-size': 10})
```
I stumbled across this limitation, and surprisingly I wasn't able to find a solution. If the feature is already implemented it would be great if anyone could share a code snippet.

Thanks!

**Comment by DougRzz** [comment]
Yes, I also could not find a way of doing this.   A label_style attribute similar to the tick_style would be nice
```python
axis.tick_style  = 
    dict({'font-family': 'Verdana', 
    'font-size': '15px',
    'font-weight': 'bold',
    'font-style': 'italic'})
```

**Comment by jcoleman42** [comment]
I came here looking for a solution to this.
